recorder.go

```diff
@@ -244,6 +244,23 @@ func GetLightStepAccessToken(lsTracer ot.Tracer) (string, error) {
 	}
 }
 
+type Closer interface {
```

Comment:
Any reason to export this interface? it's the same as `io.Closer` (which could be asserted bellow) and it's not exposed anywhere in the public API, making it look like an orphan type.